src/custom/state/orders/updaters/UnfillableOrdersUpdater.ts

```diff
@@ -34,14 +36,17 @@ async function _getOrderPrice(chainId: ChainId, order: Order) {
     chainId,
     amount,
     kind: order.kind,
+    sellToken: order.sellToken,
+    buyToken: order.buyToken,
     baseToken,
     quoteToken,
     fromDecimals: order.inputToken.decimals,
     toDecimals: order.outputToken.decimals,
+    validTo: Math.ceil(Date.now() / 1000 + 3000),
```

Comment:
It looks like it's correct, but I have 2 suggestions:

1. Magic number 3000. What's that for, and why 3000?
2. I would add 3000 after the `ceil` method, to be more explicit. After all, you are adding an integer and there is no need for rounding that

Bonus: create a helper function for that

Reply:
Actually, you can use `calculateValidTo`, right?

Reply:
order.validTo should be fine here @alfetopito